tools/ios/MASTG-TOOL-0039.md

```diff
@@ -26,9 +26,39 @@ For a quick start you can go through the [iOS examples](https://www.frida.re/doc
 
 ## Installing Frida on iOS
 
-To connect Frida to an iOS app, you need a way to inject the Frida runtime into that app. This is easy to do on a jailbroken device: just install `frida-server` through Cydia. Once it has been installed, the Frida server will automatically run with root privileges, allowing you to easily inject code into any process.
+To connect Frida to an iOS app, you need a way to inject the Frida runtime into that app. This is easy to do on a jailbroken device since we can install `frida-server` through Sileo. Start Sileo and add Frida's repository by navigating to **Manage** -> **Sources** -> **Edit** -> **Add** and entering <https://build.frida.re>. You should then be able to find and install the Frida package.
```

Comment:
```suggestion
To connect Frida to an iOS app, you need a way to inject the Frida runtime into that app. This is easy to do on a jailbroken device since you can install `frida-server` through a third-party app store such as @MASTG-TOOL-0064. Open Sileo and add Frida's repository by navigating to **Manage** -> **Sources** -> **Edit** -> **Add** and entering <https://build.frida.re>. You should then be able to find and install the Frida package.
```